<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>686</width>
    <height>695</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Checkers</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-Damka-Desktop_Qt_5_8_0_MinGW_32bit-Debug/checkers_logo.png</normaloff>../build-Damka-Desktop_Qt_5_8_0_MinGW_32bit-Debug/checkers_logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="BoardGraphicsView" name="boardGraphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>692</width>
      <height>642</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionStart_New_Game"/>
    <addaction name="actionConfiguration"/>
    <addaction name="actionCustomized_Game"/>
    <addaction name="separator"/>
    <addaction name="actionUndo_step"/>
    <addaction name="actionRedo_step"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionRules"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart_New_Game">
   <property name="text">
    <string>Start New Game</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUndo_step">
   <property name="text">
    <string>Undo step     (Ctrl+Z)</string>
   </property>
  </action>
  <action name="actionRedo_step">
   <property name="text">
    <string>Redo step     (Ctrl+Y)</string>
   </property>
  </action>
  <action name="actionRules">
   <property name="text">
    <string>Rules</string>
   </property>
  </action>
  <action name="actionCustomized_Game">
   <property name="text">
    <string>Customized Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BoardGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">boardgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
